executor/src/witgen/mod.rs

```diff
@@ -96,6 +96,8 @@ impl<'a, 'b, T: FieldElement> WitnessGenerator<'a, 'b, T> {
     /// Generates the committed polynomial values
     /// @returns the values (in source order) and the degree of the polynomials.
     pub fn generate(self) -> Vec<(String, Vec<T>)> {
+        return self.external_witness_values;
```

Comment:
Witgen fails if we pass an invalid external witness. But since we pass all witness columns anyway, there is no reason to run it in the first place.